react/tsconfig.json

```diff
@@ -1,12 +1,10 @@
 {
   "extends": "@vtex/tsconfig",
   "compilerOptions": {
+    "lib": ["es2017", "dom", "es2018.promise", "esnext.asynciterable"],
```

Comment:
Why is this different from the default?

Reply:
I was having a very annoying issue trying to use types from the `dom`. Specifically, `ChangeEvent<HTMLInputElement>` from https://github.com/vtex-apps/store-newsletter/blob/feature%2Ffirst-implementation/react/FormEmailInput.tsx#L32 was throwing me an error saying that `value` does not exist in an `HTMLInputElement`... my guess is that it was not loading the `dom` types and trying to use the ones from `@types/react`, which are wrong.